sklearn/utils/estimator_checks.py

```diff
@@ -1191,6 +1192,58 @@ def check_estimators_pickle(name, estimator_orig):
         assert_allclose_dense_sparse(result[method], unpickled_result)
 
 
+def check_pairwise_estimator_tag(name, estimator_orig):
+    attributes_to_check = [('metric', 'precomputed'),
```

Comment:
I think the code would be clearer if you just put 'precomputed' inline below

Reply:
I think it would be unfortunate if we found ourselves introducing a different placeholder!

Reply:
Good idea :) resolved!